src/lib/date-picker/base/base-date-picker-adapter.ts

```diff
@@ -56,7 +56,7 @@ export interface IBaseDatePickerAdapter extends IBaseAdapter {
   setCalendarLocale(locale: string | undefined): void;
   propagateCalendarKey(evt: KeyboardEvent): void;
 }
-export abstract class BaseDatePickerAdapter<T extends BaseComponent> extends BaseAdapter<T> implements IBaseDatePickerAdapter {
+export abstract class BaseDatePickerAdapter<T extends IBaseComponent> extends BaseAdapter<T> implements IBaseDatePickerAdapter {
```

Comment:
We only needed this changed due to the `BaseComponent` change above for `connectedCallback`. Is that change critical to this changeset?